### DISCUSSION TITLE
add logo on top and text footer

### DISCUSSION THREAD

**Discussion by SimonChiu930** [rejetto/hfs#753] [Q&A] [answered]
Add LOGO image on top of header and some text on footer, but the style want to fit the default location and center.
What CSS enter on custom HTML?

**Comment by rejetto** [comment]
it depends on what you want to get.
your request is vague.

**Comment by PH5HP** [comment]
Hi,

Something like this?

![image](https://github.com/user-attachments/assets/a318bd6b-7a4c-4124-b78c-6d88e4254900)

**Reply by SimonChiu930** [reply]
Yes, header and footer content is fit the center and scale large and small will auto

![WeChat Screenshot_20240927091345](https://github.com/user-attachments/assets/8606f7e3-deac-43bc-b38a-0afddcbc205b)

**Reply by PH5HP** [reply]
Hi,

Tip: If you are for example on my webserver page with Google Chrome as browser then you can press F12 to inspect all kind of things. Use elements in the top bar of that windows to display the HTML code. That way you can exactly see what they have done to make thet layout to look that way.

Feel free to ask...No problem...

**Reply by SimonChiu930** [reply]
Thank you very much.

I have another problem is for folder permission, but I consider how can ask, maybe open new discussion.

**Reply by PH5HP** [reply]
I would open another discussion ... This makes all more clear.

**Comment by rejetto** [comment]
the most "clear" way to center things, is to wrap them like this
```html
<center>
hello!
</center>
```

**Reply by SimonChiu930** [reply]
how about image? is it using external URL call ?

**Comment by rejetto** [comment]
for image is the same.
you can use external or internal url.

**Reply by SimonChiu930** [reply, marked as answer]
Let say, I set the Home disk source is C:\Web <~this is a web location

**Custom HTML > Before Header, add:**
`<p style="text-align:center;"><img src="/images/logo.png" width=50%></p>`
 
![image](https://github.com/user-attachments/assets/668835de-5c62-4a66-b01a-12e804d98586)

**Comment by rejetto** [comment]
i guess this is not a question, just showing that it works

**Reply by SimonChiu930** [reply]
I search HTML code on internet, before not success
thanks

**Comment by PH5HP** [comment]
Hi,

This is my header regarding what can be seen. I have this part of code in the 'Before Header' section of 'Custom HTML' in the MAIN MENU of HFS. The classes in the div are CSS code in the 'STYLE' section of 'Custom HTML'. I hope the image gives you an idea how the use your own images in the header. The CSS is not important for that. I made an 'image' drirectory in the main HFS directory and all the images I use are in it.

![image](https://github.com/user-attachments/assets/d7e3045b-898b-4d9d-8fe6-b738e62ec59e)

**Reply by SimonChiu930** [reply]
Thanks you very much
